README.md

```diff
@@ -145,6 +145,16 @@ This image is configured with volumes at `/data` and `/datalog` to hold the Zook
 
 > Be careful where you put the transaction log. A dedicated transaction log device is key to consistent good performance. Putting the log on a busy device will adversely affect performance.
 
+## How to configure logging
+
+By default, ZooKeeper redirects stdout/stderr outputs to the console. You can redirect to a file located in /logs by passing environment variable ZOO_LOG4J_PROP as follows:
+
+    $ docker run --name some-zookeeper --restart always -d -v -e ZOO_LOG4J_PROP="INFO,ROLLINGFILE" 31z4/zookeeper
+
+> This will write logs to `/logs/zookeeper.log`. Check [ZooKeeper Outputs](https://github.com/apache/zookeeper/blob/release-3.4.13/bin/zkServer.sh#L141) and [log4j.properties](https://github.com/apache/zookeeper/blob/release-3.4.13/conf/log4j.properties) for more details.
```

Comment:
Not sure if it's a good idea to refer to the source code of the specific version because the image supports multiple Zookeeper verisions.

Reply:
Yes, it's a better way. Done.